python/bayesmixpy/build_bayesmix.py

```diff
@@ -0,0 +1,22 @@
+import os
+import pathlib
+import subprocess
+
+HERE = os.path.dirname(os.path.realpath(__file__))
+path = pathlib.Path(HERE)
+BAYESMIX_HOME = path.resolve().parents[1]
+
+def build_bayesmix(nproc):
```

Comment:
I would add a message at the beginning that specifies that the library is being built